src/coreclr/jit/llvm.cpp

```diff
@@ -64,6 +65,8 @@ extern "C" DLLEXPORT void registerLlvmCallbacks(void*       thisPtr,
         _module = new Module(llvm::StringRef("netscripten-clrjit"), _llvmContext);
         _module->setTargetTriple(triple);
         _module->setDataLayout(dataLayout);
+        // TODO: better to get this from compiler, but where is it?
```

Comment:
I don't think we have a dynamic value for it. Forms I've personally seen in the past: `TARGET_POINTER_SIZE` define, `genTypeSize(TYP_I_IMPL)`. Seems like the define is most popular.